app/models/teacher_training_adviser/steps/start_teacher_training.rb

```diff
@@ -47,16 +46,28 @@ def formatted_value(item)
     def filter_items(items)
       items.select do |item|
         item.id == NOT_SURE_ID ||
-          item.value.to_i.between?(first_year, first_year + (NUMBER_OF_YEARS - 1))
+          item.value.to_i.between?(first_year, first_year + (number_of_years - 1))
       end
     end
 
     def first_year
       # After 6th September you can no longer start teacher training for that year.
-      include_current_year = Time.zone.today < Date.new(current_year, 9, 7)
+      include_current_year = Time.zone.today < date_to_drop_current_year
       include_current_year ? current_year : current_year + 1
     end
 
+    def number_of_years
+      Time.zone.today.between?(date_to_add_additional_year, date_to_drop_current_year - 1.day) ? 4 : 3
```

Comment:
If we're always doing a `-1` above could this just be

```suggestion
      Time.zone.today.between?(date_to_add_additional_year, date_to_drop_current_year - 1.day) ? 3 : 2
```

Reply:
Ah yeah good catch 👍